### PR TITLE
[#194] fixed input float labels

### PR DESCRIPTION
**Pull request by Pavliuk13**
Before:

After:

### CODE REVIEW

**Review by EugeneValko — APPROVED** [review]

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 1]
```diff
@@ -80,11 +108,13 @@
...
             <div class="item">
                 <div class="team-members-container">
-                    <p class="team-members-label">Team members {{addedMembers.length}}:</p>
+                    <p class="team-members-label">Team members {{ addedMembers.length }}:</p>
                     <div class="member-info-container" *ngFor="let member of addedMembers">
                         <div class="member-info">
-                            <p>{{member.name}}</p>
-                            <i class="fa-solid fa-xmark fa-lg cancel-icon" (click)="removeMemberToList(meetingForm)"></i>
+                            <p>{{ member.name }}</p>
+                            <i
```
Comment (apofis96) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: fix style x4

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 2]
```diff
@@ -36,17 +50,26 @@
...
+                                class="select-input unit-of-time"
                                 (selectionChange)="customDurationChanged(meetingForm)">
                                 <mat-option *ngFor="let unit of unitOfTime" [value]="unit">
-                                    {{unit}}
+                                    {{ unit }}
                                 </mat-option>
                             </mat-select>
                         </div>
                         <div *ngIf="customTimeControl.invalid && customTimeControl.touched">
-                            <small *ngIf="customTimeShown && customTimeControl.errors?.['required']">This field is required.</small>
```
Comment (apofis96) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: fix style x3

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 3]
```diff
@@ -2,32 +2,46 @@
...
                         <small *ngIf="meetingNameControl.errors?.['required']">This field is required.</small>
-                        <small *ngIf="meetingNameControl.errors?.['minlength']">Length must be more than 2
-                            characters.</small>
-                        <small *ngIf="meetingNameControl.errors?.['maxlength']">Length must be less than 50
-                            characters.</small>
+                        <small *ngIf="meetingNameControl.errors?.['minlength']"
+                            >Length must be more than 2 characters.</small
+                        >
+                        <small *ngIf="meetingNameControl.errors?.['maxlength']"
+                            >Length must be less than 50 characters.</small
```
Comment (apofis96) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: fix style x2

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 4]
```diff
@@ -2,32 +2,46 @@
...
+                        class="select-input" />
                     <div *ngIf="meetingNameControl.invalid && meetingNameControl.touched">
                         <small *ngIf="meetingNameControl.errors?.['required']">This field is required.</small>
-                        <small *ngIf="meetingNameControl.errors?.['minlength']">Length must be more than 2
-                            characters.</small>
-                        <small *ngIf="meetingNameControl.errors?.['maxlength']">Length must be less than 50
-                            characters.</small>
+                        <small *ngIf="meetingNameControl.errors?.['minlength']"
+                            >Length must be more than 2 characters.</small
+                        >
```
Comment (apofis96) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: fix style

**Review by apofis96 — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html (hunk 1 above)
Comment (Pavliuk13) [reply frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: All fixed

**Review by apofis96 — APPROVED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**